### PR TITLE
969/slice widget tests

### PR DESCRIPTION
**Pull request by danigo99**
## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: cypress/e2e/slices/03-build-slices.cy.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+    slicePage.openEditWidgetModal("ContentRelationshipField");
+    contentRelationshipModal
+      .editLabel("NewContentRelationshipField")
+      .editApiId("ContentRelationshipApiID")
+      .addCustomType(customTypeName)
+      .save();
+    slicePage.getWidgetField("NewContentRelationshipField");
+
+    slicePage.save();
```
Comment (MarcMcIntosh) [inline cypress/e2e/slices/03-build-slices.cy.js:201]: Are any assertions being made in this test?

File: cypress/e2e/slices/03-build-slices.cy.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+} from "../../pages/slices/editWidgetModals";
+
+const SLICE = {
+  id: "sliceBuild",
+  name: "SliceBuilding",
+  library: "slices",
+};
+
+describe("I am a new SM user (with Next) who wants to build a slice with different widgets.", () => {
+  let slicePage = new SlicePage();
```
Comment (MarcMcIntosh) [inline cypress/e2e/slices/03-build-slices.cy.js:26]: neat

File: cypress/pages/slices/editWidgetModals/baseEditWidgetModal.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+export class BaseEditWidgetModal {
```
Comment (MarcMcIntosh) [inline cypress/pages/slices/editWidgetModals/baseEditWidgetModal.js:1]: note to self: three assertions being made here

File: cypress/pages/slices/editWidgetModals/geoPointModal.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import { BaseEditWidgetModal } from "./baseEditWidgetModal";
+
+export class GeoPointModal extends BaseEditWidgetModal {}
```
Comment (MarcMcIntosh) [inline cypress/pages/slices/editWidgetModals/geoPointModal.js:3]: how come this one is empty?

File: cypress/pages/slices/slicePage.js [hunk 5]
```diff
@@ -75,7 +75,7 @@ export class SlicePage {
    * @param {string} type Type of the widget.
    */
   addNewWidgetField(name, type) {
-    cy.contains("button", "Add a new Field").click();
```
Comment (MarcMcIntosh) [inline cypress/pages/slices/slicePage.js:78]: did this change?

File: packages/slice-machine/lib/models/common/widgets/Image/components.tsx [hunk 6]
```diff
@@ -72,7 +72,11 @@ export const AddThumbnailButton: React.FC<{
           alignItems: "center",
         }}
       >
-        <BsFillPlusSquareFill size={34} color="#5263BA" />
+        <BsFillPlusSquareFill
+          size={34}
+          color="#5263BA"
+          aria-label="Add a thumbnail"
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/models/common/widgets/Image/components.tsx:78]: aria-labels for selectors are aa good idea 👍

File: cypress/pages/slices/editWidgetModals/geoPointModal.js (hunk 4 above)
Comment (danigo99) [reply cypress/pages/slices/editWidgetModals/geoPointModal.js:3]: Doesn't add any additional functionality from the parent class it inherits

File: cypress/pages/slices/slicePage.js (hunk 5 above)
Comment (danigo99) [reply cypress/pages/slices/slicePage.js:78]: No, but for some reason addressing the button by text doesn't work when there is the second button no the screen (this happens when there are no fields in the slice)

File: packages/slice-machine/lib/models/common/widgets/Image/components.tsx (hunk 6 above)
Comment (danigo99) [reply packages/slice-machine/lib/models/common/widgets/Image/components.tsx:78]: Yep, double benefit of an accessibility check 😄

File: cypress/e2e/slices/03-build-slices.cy.js (hunk 2 above)
Comment (danigo99) [reply cypress/e2e/slices/03-build-slices.cy.js:26]: Love this from @mdeclercq !!

File: cypress/pages/slices/slicePage.js (hunk 5 above)
Comment (MarcMcIntosh) [reply cypress/pages/slices/slicePage.js:78]: ah that makes sense

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**